src/infrastructure/NodeService.js

```diff
@@ -311,13 +311,26 @@ class NodeService {
 			throw Error('Statistics service endpoint is not provided');
 	}
 
-	static getNodePayouts = async (pageInfo, nodeId, filter) => {
+	static getNodePayouts = async (pageInfo, nodeId, filter, rewardProgram) => {
 		const endpoint = globalConfig.endpoints.nodeRewardsController;
 
 		if (endpoint && endpoint.length) {
 			const params = { pageNumber: pageInfo.pageNumber, nodeId };
 
-			let route = filter === 'voting' ? '/votingPayouts' : '/payouts';
+			let route = '/payouts';
+
+			switch (rewardProgram) {
+			case 'EarlyAdoption':
+				route = '/payouts/earlyadoption';
+				break;
+			case 'Ecosystem':
+				route = '/payouts/ecosystem';
+				break;
+			case 'SuperNode':
```

Comment:
Do we still need this case `SuperNode`? seem it already handle by default.

Reply:
For better code understanding :)